datarobot_batch_scoring/main.py

```diff
@@ -252,12 +254,18 @@ def main(argv=sys.argv[1:]):
     skip_row_id = parsed_args['skip_row_id']
     output_delimiter = parsed_args.get('output_delimiter')
 
+    if pid and not lid:
```

Comment:
That's was my thoughts. However I don't know how much effort that would be with state of code that we have, so that's why I didn't proposed it. I If we would use click that would be much easier... But overall I like this idea.